### ISSUE TITLE
Configure connection pool properties through VCAP_SERVICES

### ISSUE THREAD

**Issue by mgvinuesa** [cloudfoundry-community/tomee-buildpack-resource-configuration#8]
Hi,

Is it possible to configure the connection pool (maxActive, maxIdle...) by VCAP_SERVICES properties? Looking the code I think that currently it is not possible but maybe you have thought to solve it in the future.

If not, I suppose that the only way is going to be declaring the resource in the WEB-INF/resources.xml with the same id and the needed properties:

  `<Resource id='jdbc/ARQRUNDataSource' type='DataSource' initialSize='8' maxActive='8' minIdle='8' maxIdle='8'/>`

And the buildpack will add the properties provider.

[tomee_resource_configuration.rb#L93](https://github.com/cloudfoundry-community/tomee-buildpack/blob/master/lib/java_buildpack/container/tomee/tomee_resource_configuration.rb#L93)

Regards,
Manuel.

**Comment by violetagg** [comment]
> Looking the code I think that currently it is not possible but maybe you have thought to solve it in the future.

Yes that's correct. Currently if you provide these properties with `VCAP_SERVICES` they will not be read.

> I suppose that the only way is going to be declaring the resource in the WEB-INF/resources.xml with the same id and the needed properties

Yes

If you are interested in providing this feature. It should be done here
https://github.com/cloudfoundry-community/tomee-buildpack-resource-configuration/blob/master/src/main/java/org/cloudfoundry/reconfiguration/tomee/provider/RelationalServicePropertiesProvider.java

Regards,
Violeta
